File: Example/AzureDataAndroidExample_Java/app/src/main/java/com/azure/mobile/azuredataandroidexample_java/model/MyDocument.java
package com.azure.mobile.azuredataandroidexample_java.model;

import com.azure.data.model.Document;
import com.azure.data.model.User;
import java.util.Date;

/**
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License.
 */
public class MyDocument extends Document {

    String testString = "My Custom String";
    Integer testNumber = 0;
    Date testDate = new Date();
    Boolean testBool = false;
    int[] testArray = {1, 2, 3};
    User testObject = null;
}

File: Example/AzureDataAndroidExample/app/src/main/java/com/azure/mobile/azuredataandroidexample/model/MyDocument.kt
package com.azure.mobile.azuredataandroidexample.model

import com.azure.data.model.Document
import com.azure.data.model.User
import java.util.*

/**
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License.
 */

class MyDocument (id: String? = null) : Document(id) {

    var testString = "My Custom String"
    var testNumber = 0
    var testDate: Date = Date()
    var testBool = false
    var testArray = arrayOf(1, 2, 3)
    var testObject: User? = null
}